index.js

```diff
@@ -154,6 +155,14 @@ function resolve (queries, context) {
   }, [])
 }
 
+function checkLatestVersion (releaseDate) {
```

Comment:
If we show a warning only in Node.js, why do we need this code in non-Node.js Browserslist? ;) Just move everything to `node.js` to keep Browseslist small in browser build.